setup.py

```diff
@@ -359,7 +360,7 @@ def check_file_at_path(path2file):
 build_requires = ['numpy >={}'.format(min_numpy_build_version)]
 install_requires = [
     'llvmlite >={},<{}'.format(min_llvmlite_version, max_llvmlite_version),
-    'numpy >={}'.format(min_numpy_run_version),
+    'numpy >={},<={}'.format(min_numpy_run_version, max_numpy_run_version),
```

Comment:
Think this needs to be `<1.21` so that e.g. `1.20.3` works ok?

Reply:
oh yes, because `<=1.20` means `<=1.20.0`

Reply:
fixed in 77f3b